e2e-tests/pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>dev.streamx.cli</groupId>
+    <artifactId>streamx-cli</artifactId>
+    <version>0.3.0</version>
+  </parent>
+
+  <artifactId>streamx-cli-e2e-tests</artifactId>
+
+  <name>StreamX Cli : E2E TESTS</name>
```

Comment:
My bad. I meant not uppercase but capitalised...

So here: `StreamX Cli : E2e Tests`